homeWork2/returnUniqueNumbers.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+function returnUniqueNumbers(arrOfNumbers) {
+  return arrOfNumbers.reduce((arrUniqueOfNumbers, number) => {
+    if (!arrUniqueOfNumbers.find((uniqueNumber) => uniqueNumber === number)) {
+      arrUniqueOfNumbers = [...arrUniqueOfNumbers, number];
```

Comment:
1. check the vide I sent about reduce and `...` operator inside of it.
2. super heavy solution. O(n^2). For tasks like this the best approach is to use `Set`